src/strategies/zrx-voting-power/index.ts

```diff
@@ -0,0 +1,83 @@
+import { BigNumber } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+import fetch from 'cross-fetch';
+
+import { strategy as erc20BalanceOfStrategy } from '../erc20-balance-of';
+
+export const author = 'benlyaunzon';
+export const version = '0.1.0';
+
+const ZRX_STAKING_POOLS = {
+  '1': 'https://api.0x.org/staking/pools',
+  '42': 'https://staging.api.0x.org/staking/pools'
+};
+
+const abi = [
+  'function getVotingPower(address account, bytes32[] operatedPoolIds) view returns (uint256 votingPower)'
+];
+
+const encodePoolId = (poolId: number) =>
+  `0x${poolId.toString(16).padStart(64, '0')}`;
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag =
+    typeof snapshot === 'number'
+      ? snapshot
+      : await provider.getBlockNumber(snapshot);
```

Comment:
You don't need to query for current block number here, 'latest' will work.
```suggestion
      : 'latest';
```